test/lib/ufe/testPythonWrappers.py

```diff
@@ -82,28 +83,71 @@ def testWrappers(self):
-        else:
-            capsulePrim = mayaUsd.ufe.ufePathToPrim('|world%s,/Capsule1' % proxyShape)
+        capsulePrim = mayaUsd.ufe.ufePathToPrim('%s,/Capsule1' % proxyShape)
         self.assertIsNotNone(capsulePrim)
 
-        if ufeUtils.ufeFeatureSetVersion() >= 2:
-            # Test the maya-usd getPrimFromRawItem() wrapper.
-            capsulePath = proxyShapePath + usdUtils.createUfePathSegment('/Capsule1')
-            capsuleItem = ufe.Hierarchy.createItem(capsulePath)
-            rawItem = capsuleItem.getRawAddress()
-            capsulePrim2 = mayaUsd.ufe.getPrimFromRawItem(rawItem)
-            self.assertIsNotNone(capsulePrim2)
-            self.assertEqual(capsulePrim, capsulePrim2)
-
-            # Test the maya-usd getNodeTypeFromRawItem() wrapper.
-            nodeType = mayaUsd.ufe.getNodeTypeFromRawItem(rawItem)
-            self.assertIsNotNone(nodeType)
-
-            # Test the maya-usd getNodeNameFromRawItem() wrapper.
-            nodeName = mayaUsd.ufe.getNodeNameFromRawItem(rawItem)
-            self.assertIsNotNone(nodeName)
+        # Test maya-usd usdPathToUfePathSegment wrapper.
+        ufePath = mayaUsd.ufe.usdPathToUfePathSegment('/Capsule1')
+        self.assertEqual(ufePath, str(usdUtils.createUfePathSegment('/Capsule1')))
+
+        # Test the uniqueName wrapper.
+        # Note: param1 = list of existing children names.
+        #       param2 = source name that you want made unique
+        #       return = unique name from input source name
+        newName = mayaUsd.ufe.uniqueName([], 'Capsule')
+        self.assertEqual(newName, 'Capsule1')
+        newName = mayaUsd.ufe.uniqueName([], 'Capsule1')
+        self.assertEqual(newName, 'Capsule2')
+        newName = mayaUsd.ufe.uniqueName(['Cone1'], 'Capsule1')
+        self.assertEqual(newName, 'Capsule2')
+        newName = mayaUsd.ufe.uniqueName(['Capsule1'], 'Capsule1')
+        self.assertEqual(newName, 'Capsule2')
+        newName = mayaUsd.ufe.uniqueName(['Capsule1', 'Capsule5'], 'Capsule1')
+        self.assertEqual(newName, 'Capsule2')
+        newName = mayaUsd.ufe.uniqueName(['Capsule001'], 'Capsule001')
+        self.assertEqual(newName, 'Capsule002')
+        newName = mayaUsd.ufe.uniqueName(['Capsule001', 'Capsule002'], 'Capsule001')
+        self.assertEqual(newName, 'Capsule003')
+        newName = mayaUsd.ufe.uniqueName(['Capsule001', 'Capsule005'], 'Capsule001')
+        self.assertEqual(newName, 'Capsule002')
+
+        # Test uniqueChildName wrapper.
+        newName = mayaUsd.ufe.uniqueChildName(capsulePrim, 'Cone1')
+        self.assertEqual(newName, 'Cone1')
+        mayaUsdStage.DefinePrim("/Capsule1/Cone1", "Cone")
+        newName = mayaUsd.ufe.uniqueChildName(capsulePrim, 'Cone1')
+        self.assertEqual(newName, 'Cone2')
+        mayaUsdStage.DefinePrim("/Capsule1/Sphere001", "Sphere")
+        newName = mayaUsd.ufe.uniqueChildName(capsulePrim, 'Sphere001')
+        self.assertEqual(newName, 'Sphere002')
```

Comment:
New tests to cover uniqueName/uniqueChildName, especially the missing numerical padding case you caught with my original code.